usr/share/rear/lib/bootloader-functions.sh

```diff
@@ -529,6 +529,58 @@ function get_root_disk_UUID {
     ( set -o pipefail ; mount | grep ' on / ' | awk '{print $1}' | xargs blkid -s UUID -o value || Error "Failed to get root disk UUID" )
 }
 
+# Detect whether actually GRUB 2 is installed and that test is to
+# check if grub-probe or grub2-probe is installed because
+# grub-probe or grub2-probe is only installed in case of GRUB 2.
+# Needed because one can't tell the GRUB version by looking at the MBR
+# (both GRUB 2 and GRUB Legacy have the string "GRUB" in their MBR).
+function is_grub2_installed () {
+    if type -p grub-probe >&2 || type -p grub2-probe >&2 ; then
+        LogPrint "GRUB 2 is installed (grub-probe or grub2-probe exist)."
+        return 0
+    else
+        return 1
+    fi
+}
+
+# Determine whether a disk is worth detecting or installing GRUB on
+function is_disk_grub_candidate () {
+    local disk="$1"
+    local disk_partitions part
+    local label flags
+
+    # ToDo : validate $disk (does it even exist? Isn't it write-protected?)
+
+    # Installing grub on an LVM PV will wipe the metadata so we skip those
+    is_disk_a_pv "$disk" && return 1
+
+    label="$( get_disklabel_type "$disk" )" || return 1
+    # I (pcahyna) don't know whether the SUSE-specific 'gpt_sync_mbr' partition scheme
+    # supports GRUB installation without a bios_grub partition.
```

Comment:
@jsmeix Thanks for the explanation. The code currently does not have any `gpt_sync_mbr` handling, the comment explains why it is not needed. So the only thing I could improve is to delete the comment to avoid distracting the reader from more important stuff.

Reply:
done in [acff60f](https://github.com/rear/rear/pull/3145/commits/acff60fdb7f4943991f1dbd952547fa6076b3ab6)